tests/scalers/github_runner/github_runner_test.go

```diff
@@ -0,0 +1,337 @@
+//go:build e2e
+// +build e2e
+
+package github_runner_test
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/google/go-github/v50/github"
+	"github.com/joho/godotenv"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/oauth2"
+	"k8s.io/client-go/kubernetes"
+
+	. "github.com/kedacore/keda/v2/tests/helper"
+)
+
+// Load environment variables from .env file
+var _ = godotenv.Load("../../.env")
+
+const (
+	testName = "github-runner-test"
+)
+
+var (
+	personalAccessToken = os.Getenv("GITHUB_PAT")
```

Comment:
You can use `GH_AUTOMATIONS_PAT`. It has permissions to manage the runners pool

Reply:
on it ;p